### ISSUE TITLE
Delayed client mouse and keyboard update after short idle time

### ISSUE THREAD

**Issue by axlan** [symless/synergy#6061]
### Operating Systems ###

Server: Ubuntu 17.04
Client: Windows 10 Pro Version 1703

### Synergy Version ###

1.8.8-stable

### Steps to reproduce bug ###

Using a wireless Microsoft Sculpt Comfort Desktop (L3V-00001) mouse/keyboard set

On the client leave the mouse and keyboard idle for a second or so then try to move the mouse or keyboard. About 25% of the time the input will be delayed by a couple seconds then occur all at once. This can even result in the mouse reappearing in the Server's screen without being updated on the client. This makes it appear that the mouse input is being registered, but it's failing to make it to the client.

Possibly a duplicate of https://github.com/symless/synergy/issues/3807 or https://github.com/symless/synergy/issues/6055

**Comment by axlan** [comment]
Tried https://symless.com/forums/topic/2587-download-synergy-190-to-fix-the-wifi-macos-bug/ without seeing any improvement

**Comment by axlan** [comment]
Confirmed by watching the logs that the server gets the input, but there's a large delay to getting to the client, but only after idle periods

**Comment by axlan** [comment]
Issue doesn't appear if I swap server/client roles

**Comment by XinyuHou** [comment]
@axlan 
Thank you for raising this issue.

Could you double check if the delay is caused by disconnection? You can have a look at the logging output from Synergy window. If a disconnection happens, there should be a line like "Client abc is disconnected"

**Comment by axlan** [comment]
My memory was that the logs did not show disconnections. When verbose logging was turned on I would see the actions appear immediately on the server, then a couple seconds later appear on the client. There was no disconnection during that period. However, I just tried switching back to this configuration to confirm and cannot reproduce the issue. I've disabled SSL and changed the DPI settings on the Windows machine in the mean time, but I'm not sure what would have fixed this issue. Closing the issue since I can no longer reproduce.
